assets/js/translator.js

```diff
@@ -6,6 +6,7 @@ var curSrcLang, curDstLang;
 var recentSrcLangs = [], recentDstLangs = [];
 var droppedFile;
 var textTranslateRequest;
+var webpageTranslateRequest;
```

Comment:
Yes, this does make sense since at any given time, only one of these requests would work! I would change it.